core/templates/pages/exploration-editor-page/editor-tab/graph-directives/state-graph-visualization.directive.ts

```diff
@@ -177,9 +177,19 @@ angular.module('oppia').directive('stateGraphVisualization', [
                 var dimensions = getElementDimensions();
 
                 // Center the graph at the node representing the current state.
-                origTranslations[0] = (
-                  dimensions.w / 2 - nodeData[$scope.currentStateId()].x0 -
-                  nodeData[$scope.currentStateId()].width / 2);
+                try {
+                  origTranslations[0] = (
+                    dimensions.w / 2 - nodeData[$scope.currentStateId()].x0 -
+                    nodeData[$scope.currentStateId()].width / 2);
+                } catch (e) {
+                  console.error(
```

Comment:
Console error would only log things in the users browser console. We want the error to be reported in our server. To do this, you will need to catch the error, append the extra details as a string to the error message and then rethrow the error -- unhandled exceptions are automatically caught and sent to the server.

E.g. 

```ts
error.message += `\ncurrentStateId(): {$scope.currentStateId()}\nnodeData: {nodeData}`;
throw error;
```